zerver/models.py

```diff
@@ -1682,12 +1681,11 @@ def to_dict(self):
     @staticmethod
     def status_from_string(status):
         # type: (NonBinaryStr) -> Optional[int]
+        status_val = None
         if status == 'active':
             status_val = UserPresence.ACTIVE
         elif status == 'idle':
             status_val = UserPresence.IDLE
-        else:
-            status_val = None
```

Comment:
@neiljp , are you referring to a version of this newer than 09cf45499? It sounds like you are but that's the latest version I see on GitHub, so just wanted to make sure if there's another push you'd meant to do. :-)